tests/test_image.py

```diff
@@ -55,11 +55,9 @@ def test_ImageWriter(tmp_path):
 
 @pytest.mark.xfail(strict=True)
 def test_ImageReader(data_path):
-    d = data_path / "images"
-    d.mkdir()
-    p = d / "test_image_3.png"
+    d = data_path / "images/test_image_3.png"
     with Pipeline() as pipeline:
-        image = ImageReader(p)
+        image = ImageReader(d)
 
     stream = pipeline.transform_stream()
     pipeline.run()
```

Comment:
It gives error saying that

> The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()

Reply:
Sorry, can you please merge again and remove the xfail?

In general, can you please commit your failing tests as they are (without `xfail`) so it is obvious that the test fail?

Reply:
I have removed the xfail. but the test_ImageReader keeps on failing